front/src/applications/stdcm/types.ts

```diff
@@ -75,19 +75,16 @@ export type StdcmResultsOperationalPoint = {
   stopRequested: boolean;
 };
 
+export type FieldError = {
+  message?: string;
+  display: boolean;
+  type?: 'missing' | 'invalid';
```

Comment:
This field is always defined I think ?
```suggestion
  type: 'missing' | 'invalid';
```

Reply:
done

Reply:
done